core/matrix/dense.cpp

```diff
@@ -266,8 +270,20 @@ void Dense<ValueType>::inv_scale_impl(const LinOp *alpha)
         GKO_ASSERT_EQUAL_COLS(this, alpha);
     }
     auto exec = this->get_executor();
-    exec->run(dense::make_inv_scale(
-        make_temporary_conversion<ValueType>(alpha).get(), this));
+    // if alpha is real (convertible to real) and ValueType complex
+    if (dynamic_cast<const ConvertibleTo<Dense<>> *>(alpha) &&
+        is_complex<ValueType>()) {
+        // use the real-complex kernel
+        exec->run(dense::make_inv_scale_real_complex(
+            make_temporary_conversion<remove_complex<ValueType>>(alpha).get(),
+            dynamic_cast<complex_type *>(this)));
```

Comment:
codcov reports it is not covered by test.
It should be checked by SingleVectorComplexRealScaleIsEquivalentToRef.

Reply:
Codecov reports the first line (277) as covered, only the last line (279) seems to be missing? I guess we can safely ignore this, maybe the line doesn't get deleted as dead code in the binary for real ValueType?